harmonica/_utils.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) 2018 The Harmonica Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# This code is part of the Fatiando a Terra project (https://www.fatiando.org)
+#
+import numpy as np
+
+
+def magnetic_ang_to_vec(intensity, inclination, declination):
+    """
+    Convert intensity, inclination and declination angles of the magnetic field to a
+    3-component magnetic vector.
+
+    .. note::
+        Inclination is measured positive downward from the horizontal plane, and
+        declination is measured with respect to north, where positive angles indicate
+        east.
+
+    Parameters
+    ----------
+    intensity: float or array
+        Intensity (norm) of the magnetic vector in degree.
+    inclination : float or array
+        Inclination angle of the magnetic vector in degree.
+        It mast be in ``degrees``.
+    declination : float or array
+        Declination angle of the magnetic vector.
+        It mast be in ``degrees``.
+
+    Returns
+    -------
+     magnetic_e : float or array
+        Easting component of the magnetic vector.
+    magnetic_n : float or array
+        Northing component of the magnetic vector.
+    magnetic_u : float or array
+        Upward component of the magnetic vector.
+
+    Examples
+    --------
+    >>> magnetic_e, magnetic_n, magnetic_u = magnetic_ang_to_vec(3.0, 45.0, 45.0)
+    >>> print(magnetic_e, magnetic_n, magnetic_u)
+    1.5, 1.5000000000000002, -2.1213203435596424
+    """
+    # Transform to radians
+    inc_rad = np.radians(inclination)
+    dec_rad = np.radians(declination)
+    # Calculate the 3 components
+    magnetic_e = intensity * np.cos(inc_rad) * np.sin(dec_rad)
+    magnetic_n = intensity * np.cos(inc_rad) * np.cos(dec_rad)
+    magnetic_u = -intensity * np.sin(inc_rad)
+    return magnetic_e, magnetic_n, magnetic_u
+
+
+def magnetic_vec_to_ang(magnetic_e, magnetic_n, magnetic_u, degrees=True):
```

Comment:
Could we increase the verbosity in these functions names? I agree that using full words make them too long, but something like this might be better:
```suggestion
def magnetic_vec_to_angles(magnetic_e, magnetic_n, magnetic_u, degrees=True):
```